airflow/providers/amazon/aws/hooks/s3.py

```diff
@@ -713,7 +713,9 @@ async def is_keys_unchanged_async(
             }
 
         if last_activity_time:
-            inactivity_seconds = int((datetime.now() - last_activity_time).total_seconds())
+            inactivity_seconds = int(
+                (datetime.now(last_activity_time.tzinfo) - last_activity_time).total_seconds()
```

Comment:
I believe the last_activity_time initial value will come from the same function 1st call. There is no harm in considering the tzinfo from the other one. I am trying to understand here.

Reply:
Yep, they are supposed to be at the same timzone